app/controllers/mixins/middleware_operations_mixin.rb

```diff
@@ -103,15 +103,17 @@ def log_operation_in_timeline(operation_info, mw_item)
     if operation_info.fetch(:log_timeline, false)
       EmsEvent.add_queue(
         'add', mw_item.ext_management_system.id,
-        :ems_id          => mw_item.ext_management_system.id,
-        :source          => 'EVM',
-        :timestamp       => Time.zone.now,
-        :event_type      => operation_info.fetch(:log_timeline),
-        :message         => _('%{server} will be %{operation} per user request') %
+        :ems_id                 => mw_item.ext_management_system.id,
```

Comment:
is this only a formatting change? it looks now not consistent with the rows above. I would not include it in this PR